doc/release-notes.md

```diff
@@ -4,13 +4,73 @@ release-notes at release time)
 Notable changes
 ===============
 
+Mnemonic Recovery Phrases
+-------------------------
+
+The zcashd wallet has been modified to support ZIP 339 (to be compatible with BIP 39)
```

Comment:
```suggestion
The zcashd wallet has been modified to support BIP 39,
```
(and optionally rewrap lines).

https://github.com/zcash/zips/issues/364#issuecomment-1036402343

Reply:
Addressed in https://github.com/zcash/zcash/pull/5543